rules/sql.go

```diff
@@ -212,8 +293,11 @@ func NewSQLStrFormat(id string, conf gosec.Config) (gosec.Rule, []ast.Node) {
 			},
 		},
 	}
-	rule.calls.AddAll("fmt", "Sprint", "Sprintf", "Sprintln", "Fprintf")
+	rule.AddAll("*database/sql.DB", "Query", "QueryContext", "QueryRow", "QueryRowContext")
+	rule.AddAll("*database/sql.Tx", "Query", "QueryContext", "QueryRow", "QueryRowContext")
+	rule.fmtCalls.AddAll("fmt", "Sprint", "Sprintf", "Sprintln", "Fprintf")
 	rule.noIssue.AddAll("os", "Stdout", "Stderr")
 	rule.noIssueQuoted.Add("github.com/lib/pq", "QuoteIdentifier")
```

Comment:
Why is here PostgreSQL import? It should be DB independent, right?

Reply:
This package seems to be used by quite some people. It was already in there. I don't want to remove it as part of this refactoring since it appears to be useful to remove some false positives.

Reply:
Do you feel this approach is a good practice? IMHO when the PostgreSQL hide some false alarms doesn't mean that for instance MySQL users can benefit from this.